### PR TITLE
Advanced_html_css ,the_web_content_accessibility_guidelines_wcag.md:

### PR DESCRIPTION
**Pull request by RaGuL2k23**
Fixed blank additional resources section to
say help expand this section on github

## Because
Additional resources sections seems to point out blank links
without any links specified to it.

## This PR
* Removed sentence pointing to nowhere and 
* Asks readers to expand the section on github

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 1]
```diff
@@ -58,4 +58,4 @@ This section contains questions for you to check your understanding of this less
 
 ### Additional resources
 
-This section contains helpful links to other content. It isn’t required, so consider it supplemental.
+Expand this section to have more resources on github
```
Comment (thatblindgeye) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:61]: When a lesson has an additional resources section that doesn't contain any resources, it must follow our styleguide and contain the following content:

```markdown
This section contains helpful links to related content. It isn’t required, so consider it supplemental.

- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Change needed below. Additionally there are other errors being flagged by our linter. If you'd like to resolve them feel free to, but if not then no worries

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md (hunk 1 above)
Comment (RaGuL2k23) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:61]: ok sir
i will update as you said

**Review by thatblindgeye — COMMENTED** [review]
@RaGuL2k23 thanks for the update! Would you be interested in making additional updates to the errors our linter has flagged? No pressure, though!

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md (hunk 1 above)
Comment (RaGuL2k23) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md]: i will try to correct the linting errors

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md [hunk 2]
```diff
@@ -4,14 +4,16 @@ After the previous lesson, you should now have a better understanding of how imp
 
 Another source, one that goes over many different ways to help make websites more accessible, is the Web Content Accessibility Guidelines (WCAG). In this lesson we're going to skim the surface of the WCAG, just to familiarize you with them at a more basic level.
 
-### Learning outcomes
+### Lesson overview;
```
Comment (thatblindgeye) [inline advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:7]: For this error, the message may not be super great. The `Expected: ### Lesson overview;` is meant to be read more as `Expected: "### Lesson overview";`, so we want to omit the semi-colon here.

**Review by thatblindgeye — COMMENTED** [review]
Thanks for the additional updates! Still a few linting errors being flagged (some might be just due to an extra whitespace somewhere). The way the linter works seems to be that some rules may not be flagged until other rules are resolved. Let me know if you'd mind resolving the remaining errors

**Comment by RaGuL2k23** [comment]
Now the ordered list lint error somehow magically gone . 
It looks like lint error free , but idk.🤷‍♂️

**Comment by thatblindgeye** [comment]
@RaGuL2k23 I think part of the issue may be that when you push we have to approve running the checks since you're a new contributor, plus the fact that some of the linting errors only show up when other errors are resolved. If you'd want I can merge this as is and we'll get the other errors resolved in a followup. Unless you want to make additional updates. No pressure either way, though!

**Comment by RaGuL2k23** [comment]
Ok i'll solve other lint errors if specified (to my best ill try ) .
As it helps me learn and gain knowledge

**Comment by RaGuL2k23** [comment]
And "Ordered list item prefix"  lint error is appearing even though everything is perfectly ordered

**Comment by thatblindgeye** [comment]
@RaGuL2k23 this is looking great! For the ordered list error, while the ordered lists are numbered correctly (1, 2, 3...), the linter is expecting a "lazy ordered list", where each number prefix is `1.`. So for example:

```
1. One
1. Two
1. Three
```
Would render correctly as:

1. One
1. Two
1. Three

**Review by thatblindgeye — APPROVED** [review]
Aye we got there @RaGuL2k23! Thanks for these updates again!

**Comment by RaGuL2k23** [comment]
Thanks for helping me throughout this thankyou sir.

**Pull request merged**